test/lib/edamam_wrapper_test.rb

```diff
@@ -1,82 +1,36 @@
-        "2 to 3 heads cauliflower"
-      ],
-      "calories": 1366.364111111111,
-      "totalNutrients": {
-        "ENERC_KCAL": {
-          "label": "Energy",
-          "quantity": 948.5783323955536,
-          "unit": "kcal"
-        },
-        "FAT": {
-          "label": "Fat",
-          "quantity": 66.042283310318,
-          "unit": "g"
-        }
-      }
-    }
+describe EdamamWrapper do
+  describe 'search_recipes' do
+    it "gives a list of recipes" do
+      VCR.use_cassette('recipes') do
+        recipes = EdamamWrapper.search_recipes("brains")
 
-    @recipe = Recipe.new(data[:label],
-      data[:uri],
-      data[:url],
-      data[:source],
-      options = {
-        image: data[:image],
-        ingredients: data[:ingredientLines],
-        calories:data[:calories]}
-      )
-    end
-  describe "initialize" do
-      it "can be created with valid data" do
-        @recipe.must_be_kind_of Recipe
-        @recipe.name.must_equal "Pickled Brains recipes"
-        @recipe.uri.must_equal "http://www.edamam.com/ontologies/edamam.owl#recipe_3c5cf4968fa095cbdd9fca6ab5389e9c"
-        @recipe.image.must_equal "https://www.edamam.com/web-img/811/811a71bad21d73adb7fc6e92801f8ec4"
-        @recipe.ingredients.must_equal [
-          "1 cup red-wine vinegar",
-          "3 cups white-wine vinegar",
-          "1 cup sugar",
-          "1/2 cup salt",
-          "1 tablespoon black peppercorns",
-          "1 tablespoon brown mustard seeds",
-          "5 or 6 red pearl onions",
-          "2 to 3 heads cauliflower"
-        ]
-        @recipe.calories.must_equal 1366.364111111111
-        @recipe.link.must_equal "http://www.marthastewart.com/318853/pickled-brains"
-        @recipe.source.must_equal "Martha Stewart"
+        recipes.each do |recipe|
+          recipe.must_be_kind_of Recipe
+        end
       end
+    end
+  end
 
-      it "raises error if data is incomplete" do
-        proc { Recipe.new() }.must_raise ArgumentError
-        proc { Recipe.new("uri", "label") }.must_raise ArgumentError
-        proc { Recipe.new("", "", "", "") }.must_raise ArgumentError
-      end
+  describe 'get_recipe' do
+    it "retrieves a specific recipe" do
+      VCR.use_cassette('recipes') do
+        recipes = EdamamWrapper.search_recipes("brains")
+        recipe = recipes.first
 
-      it "raises error if required data contain nil" do
-        proc { Recipe.new("name", nil, "image", "ingredients", "calories", "link", "source") }.must_raise ArgumentError
+        first_recipe = EdamamWrapper.get_recipe(recipe.id)
+        first_recipe.must_be_kind_of Recipe
+        first_recipe.name.must_equal recipe.name
       end
     end
-  describe "id" do
-      it "must return the id of a uri" do
-        @recipe.id.must_equal "3c5cf4968fa095cbdd9fca6ab5389e9c"
+
+    it "raises StandardError for a bad recipe id" do
+      VCR.use_cassette('recipes') do
+        proc { EdamamWrapper.get_recipe(0000) }.must_raise
```

Comment:
You should use `expect` instead of `proc`, that behavior is old-minitest and will be depreciated in the next version.